<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">HomeWindow {
	background-image : url(:/HomeWindow.png);
	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PB_profile">
    <property name="geometry">
     <rect>
      <x>954</x>
      <y>25</y>
      <width>124</width>
      <height>124</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>124</width>
      <height>124</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #PB_profile {
	background-image : url(:/icons/EditProfile.png);
	background-position : center;
	background-repeat : no repeat;
	background-size: contain;
	display:inline-block;
	border-radius:0px;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:12px;

	padding:50px 10px;
}

QPushButton:hover {
	background-color:#bbd3f4;
}
QPushButton:pressed {
	background-color:#transparent;
	color:#000000;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_logout">
    <property name="geometry">
     <rect>
      <x>954</x>
      <y>150</y>
      <width>124</width>
      <height>124</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>124</width>
      <height>124</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #PB_logout {
	background-image : url(:/icons/Logout.png);
	background-position : center;
	background-repeat : no repeat;
	background-size: contain;
	display:inline-block;
	border-radius:0px;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:12px;

	padding:50px 10px;
}

QPushButton:hover {
	background-color:#bbd3f4;
}
QPushButton:pressed {
	background-color:#transparent;
	color:#000000;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LB_welcome">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 20px;
letter-spacing: 0px;
word-spacing: 0px;
color: #2A2F4F;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
    </property>
    <property name="text">
     <string>Welcome Ryan</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>471</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeWidget {
    background-color: #fbf8fc;
    border: 1px solid #d3d3d3;
    alternate-background-color: #e3e3e3;
}

QTreeView::item {
    padding: 6px;
}

QTreeView::item:selected {
	background-color: #d6b8e1;
    color: #ffffff;
}

QTreeView::item:hover {
    background-color: #ffdaea;
}

</string>
    </property>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Deadline</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="PB_done">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>600</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #6ac39d;
    border: 1px solid #70ad5e;
    border-radius: 15px;
    padding: 8px 8px;
	color: #fefeff;
	font-family : Cabin;
	font-weight : bold;
	font-size : 12px;

	
}

QPushButton::hover {
	background-color : #7ccba9;

}

QPushButton:disabled {
	background-color : #7a9a71;
	border: 1px solid #7a9a71;
}</string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QWidget" name="addSubWidget" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>330</width>
      <height>270</height>
     </rect>
    </property>
    <widget class="QPushButton" name="PB_addSub">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #735bbc;
	border-radius:20px;
	display:inline-block;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:17px;
	font-weight:bold;
	padding:10px 45px;
	text-decoration:none;
	text-shadow:0px 1px 0px #2f6627;
}

QPushButton:hover {
	background-color:#826dc3;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DE_deadlineSub">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>120</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateEdit {
    background-color: #eeebf7; 

    padding: 1px; 
	font-family: Calibri;
	font-size: 16px;
	color: #634ab3;
	font-weight: bold;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #634ab3;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_titleSub">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>180</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color: #eeebf7;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size: 12px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #634ab3;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Add Subtask</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="TE_desSub">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #eeebf7;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size: 12px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #634ab3;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Deadline</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #634ab3;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_taskTitle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 14px;
letter-spacing: 0px;
word-spacing: 0px;
color: #634ab3;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Task: title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="LB_errorSubTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ef749a;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;

</string>
     </property>
     <property name="text">
      <string>error</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>330</width>
      <height>270</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="TE_desTask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #fdfcfe;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size: 12px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 15px;
letter-spacing: 0px;
word-spacing: 0px;
color: #1a699d;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Add Task</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DE_deadlineTask">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>120</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateEdit {
    background-color: #fdfcfe; 

    padding: 1px; 
	font-family: Calibri;
	font-size: 16px;
	color: #1a699d;
	font-weight: bold;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #1a699d;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_addTask">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color:#549BFF;
	border-radius:20px;
	display:inline-block;
	cursor:pointer;
	color:#ffffff;
	font-family:Arial;
	font-size:17px;
	font-weight:bold;
	padding:10px 45px;
	text-decoration:none;
	text-shadow:0px 1px 0px #2f6627;
}

QPushButton:hover {
	background-color:#6eaaff;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #1a699d;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #1a699d;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
</string>
     </property>
     <property name="text">
      <string>Deadline</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_titleTask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>180</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color:#fdfcfe;
	border-radius:9px;
	border:1px solid #7e93ff;
	display:inline-block;
	cursor:text;
	color:#6671d4;
	font-family:Impactl;
	font-size: 12px;
	font-weight:bold;
	padding:5px 20px;
	text-decoration:none;
	text-shadow:10px 0px 0px #9752cc;
}
</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="LB_errorTaskTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Arial;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #ef749a;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;

</string>
     </property>
     <property name="text">
      <string>error</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
